doc/filters/markdown_to_html.rst

```diff
@@ -67,6 +67,12 @@ You can also use the filter on an included file or a variable:
                 }
             }
         });
-       
+
     Afterwards you need to install a markdown library of your choice. Some of them are
     mentioned in the ``require-dev`` section of the ``twig/markdown-extra`` package.
+
+.. note::
+
+    If using ``twig/extra-bundle`` with ``league/commonmark`` as your markdown library
```

Comment:
Twig documentation does not assume you are using Symfony, so this not must make it very clear that this is only for Symfony (full-stack) users.